src/main/java/com/github/martynagil/drugstoremanagement/service/ShopService.java

```diff
@@ -44,4 +48,12 @@ private Shop createShopFromDto(ShopDto shopDto) {
                 )
         );
     }
+
+    private Boolean shopExists(ShopDto shopDto) {
+        return shopRepository
```

Comment:
tutaj sprawdzanie tego też jest dosyć wątpliwe... a jak dodam sklep o innej nazwie pod ten sam adres? Trochę to nie ma sensu, też bym pominął